_posts/2022-12-02-zep1.markdown

```diff
@@ -1,36 +1,43 @@
 ---
 layout: post
-title: "State of Zarr v3 and ZEP 1"
-description: Blog Post with updates regarding the development of Zarr v3
+title: "State of Zarr V3 and ZEP0001"
+description: Blog Post with updates regarding the development of Zarr V3
 date: 2022-12-02
 categories: blog
 permalink: /zep1-update/
 
 ---
 
-The Zarr community is working on a new version of Zarr, v3, which was drafted
-and proposed via [ZEP 1] by [Alistair Miles]. [Jonathan Striebel] and
-[Jeremy Maitin-Shepard] have now taken over the authorship of [ZEP 1].
+The Zarr community is working on a new version of Zarr, V3, which was drafted
+and proposed via [ZEP0001] by [Alistair Miles]. [Jonathan Striebel] and
```

Comment:
Do you want to add a "We, ..., " here? if they are the authors?

Reply:
No objection.